org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/AbstractTypeDeclaration.java

```diff
@@ -35,7 +35,7 @@ public abstract class AbstractTypeDeclaration extends BodyDeclaration {
 	 * legal Java class identifier.
 	 * @since 2.0 (originally declared on {@link TypeDeclaration})
 	 */
-	SimpleName typeName = null;
+	volatile SimpleName typeName = null;
```

Comment:
May I ask you to remove `= null` assignments for touched fields, because 1) it us useless code and 2) it would make debugging experience better (no jumping on object creation code from field to field)

Reply:
ok, a regexp fixed that.